### PR TITLE
Edits to BigAnimal PR5479

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/biganimal/release/getting_started/managing_cluster.mdx [hunk 1]
```diff
@@ -16,7 +16,7 @@ While paused, clusters aren't upgraded or patched, but upgrades are applied when
 
 After seven days, single-node and high-availability clusters automatically resume. Resuming a cluster applies any pending maintenance upgrades. Monitoring begins again.  
 
-With CLI 3.7.0 and later, you can [pause and resume cluster using CLI](../../getting_started/managing_cluster/#pausing-a-cluster).
+With CLI 3.7.0 and later, you can [pause and resume a cluster using the CLI](../../reference/cli/managing_clusters/#pausing-a-cluster).
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/getting_started/managing_cluster.mdx:19]: This was the link to the next section, not the CLI.

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 2]
```diff
@@ -298,30 +298,31 @@ To restore a deleted cluster, use the `--from-deleted` flag in the command.
 You can restore a cluster in a single cluster to a primary/standby high-availability cluster and vice versa. You can restore a distributed high-availability cluster only to a cluster using the same architecture.
 !!!
 
-### Pausing a cluster
+### Pause a cluster
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:301]: This was not the same verb form as all the other sections of this topic

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 3]
```diff
@@ -298,30 +298,31 @@ To restore a deleted cluster, use the `--from-deleted` flag in the command.
...
 ```
 biganimal cluster pause {--id | --provider --region --name}
 ```
 
-Where `id` is a valid cluster ID. The `id` is mandatory.
-      `provider` is a cloud provider of the cluster.
-      `region` is the region of the cluster.
-      `name` is the name of the cluster.
+Where:
+- `id` is a valid cluster ID. The `id` is mandatory.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:310]: @nidhibhammar It doesn't seem that `id` is mandatory if you can instead use provider/region/name.

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 4]
```diff
@@ -298,30 +298,31 @@ To restore a deleted cluster, use the `--from-deleted` flag in the command.
...
 
-Pausing a cluster using ID:
+To pausing a cluster using the ID:
 
 ```
 biganimal cluster pause --id p-c5fh47nf
 ```
 
-Pausing a cluster using name, provider, and region:
+To pause a cluster using name, provider, and region:
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:325]:
```suggestion
To pause a cluster using the name, provider, and region:
```

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 5]
```diff
@@ -341,30 +342,31 @@ __OUTPUT__
...
 ```
 biganimal cluster resume {--id | --provider --region --name}
 ```
 
-Where `id` is a valid cluster ID. The `id` is mandatory.
-      `provider` is a cloud provider of the cluster.
-      `region` is the region of the cluster.
-      `name` is the name of the cluster.
+Where:
+- `id` is a valid cluster ID. The `id` is mandatory.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:354]: @nidhibhammar again, this doesn't seem to be mandatory if you can instead use provider/region/name.

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 6]
```diff
@@ -341,30 +342,31 @@ __OUTPUT__
...
+- `provider` is a cloud provider of the cluster.
+- `region` is the cluster region.
+- `name` is the name of the cluster.
 
-If `id` of the cluster isn't known then use `--provider --region --name` to identify the cluster.
+You don't know the `id` of the cluster, use `--provider --region --name` to identify the cluster.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:359]:
```suggestion
If you don't know the `id` of the cluster, use `--provider --region --name` to identify the cluster.
```

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 7]
```diff
@@ -582,17 +584,17 @@ The `--id` and `--group-id` flags are mandatory. For example:
...
+### Pause a distributed high-availability cluster
 
-To pause a distributed high-availability cluster, use `pgd pause` command. The `pgd pause` command supports the `flag` mode only. The syntax for the command is:
+To pause a distributed high-availability cluster, use the `pgd pause` command. The `pgd pause` command supports `flag` mode only. The syntax for the command is:
 
 ```
 biganimal pgd pause {--id}
 ```
 
 Where `id` is a valid cluster ID. The `id` flag is mandatory.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:595]:
```suggestion
Where `id` is a valid cluster ID. The `id` is mandatory.
```

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 8]
```diff
@@ -606,9 +608,9 @@ To resume a distributed high-availability cluster, use the `pgd resume` command.
 biganimal pgd resume {--id}
 ```
 
-Where, `id` is a valid cluster ID. The `id` flag is mandatory.
+Where `id` is a valid cluster ID. The `id` flag is mandatory.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:611]:
```suggestion
Where `id` is a valid cluster ID. The `id` is mandatory.
```

**Comment by ebgitelman** [comment]
@nidhibhammar Please approve so we can get this merged. Thanks!

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 3 above)
Comment (nidhibhammar) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:310]:
```suggestion
- `id` is a valid cluster ID.
```

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 5 above)
Comment (nidhibhammar) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:354]:
```suggestion
- `id` is a valid cluster ID.
```

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**